api/forms.py

```diff
@@ -19,3 +19,7 @@ class UserRegistrationForm(UserCreationForm):
         help_text=_("Enter the same password as before, for verification."),
         required=False
     )
+
+
+class SecurityUserRegistrationForm(UserRegistrationForm):
```

Comment:
Not sure the `SecurityUserRegistrationForm` class is necessary since it inherits from `UserRegistrationForm` and does not extend it in any way. 
We have two options here:
1. Set `add_form= UserRegistrationForm` in the `SecurityUserAdmin` ([here](https://github.com/AndelaOSP/art-backend/pull/60/files#diff-ee05abd68c2faa98e3f8f2ac878d3231R30)) and do away with the empty `SecurityUserRegistrationForm`
2. Create a `BaseUserRegistrationForm` that will be inherited by `UserRegistrationForm` and `SecurityUserRegistrationForm`. Each will customize it differently. This will definitely warrant more effort though.

However, since `UserRegistrationForm` is already defined and does what we want, I'd recommend option 1. #

Reply:
Alright going with option 1

Reply:
Done.